lightbeam/send.py

```diff
@@ -112,8 +109,10 @@ async def do_send(self, endpoint):
         for file_name in data_files:
             with open(file_name) as file:
                 # process each line
+                line_counter = 0
```

Comment:
Could we use enumerate() here as well?

Reply:
@jayckaiser can you elaborate what you mean? I'm not sure I understand.

Reply:
Done!